apps/src/music/views/MusicView.jsx

```diff
@@ -321,6 +322,21 @@ class UnconnectedMusicView extends React.Component {
     return this.props.isPlaying;
   };
 
+  getValidationTimeout = () => {
+    // The level can specify a desired timeout, in measures, when we can start showing
+    // non-success validation messages.  That said, if we've just completed playing the
+    // last measure before reaching that specified value, we can start showing the
+    // messages.
+    // If no timeout is specified, then we can starting showing the non-success messages
+    // at measure 2.
+    return this.props.levelData?.validationTimeout
```

Comment:
Can we define this on `levelProperties.validations`? That way it's co-located with other validation data and could be edited with the UI?

Edit: oh but `validations` is an array so it would mean a lot of code changes to update its structure. Fine with it here for now then, maybe we can move it as a follow-up.

Reply:
Good point, and agreed that it's a good follow-up.